src/services/loopd/loop-service.ts

```diff
@@ -41,6 +43,9 @@ export const LoopService = ({
   lndInstanceName,
 }: LoopdConfig): ISwapService => {
   const mac = Buffer.from(macaroon, "base64").toString("hex") as Macaroon
+  if (!tlsCert) {
```

Comment:
>probably here should be the validation, also service should not throw exceptions. Please check onchain service as example

copied from https://github.com/GaloyMoney/galoy/pull/1733#discussion_r1007112288

Reply:
This should not ever happen - either config is set and `tlsCert` is never null or it is not set and this code should never be called see https://github.com/GaloyMoney/galoy/pull/1891/commits/eab6cd41a50e4af67934fbd277726dff945fad85